client/src/redux/sagas/buildSetSaga.js

```diff
@@ -36,7 +51,10 @@ function getBuildReqQString(commits, buildData) {
 function* handleBuildsLoad() {
   try {
     const commits = yield select(state => state.commits);
-    const buildSetsRaw = yield call(fetchBuildSets);
+    const buildSetsRaw = yield call(
+      fetchBuildSets,
+      convertIsoToUnixTime(commits[0], commits[9])
```

Comment:
Are we sure that commits are sorted by date?

Reply:
If I'm not mistaken, that will cut off everything which is triggered by hand

Reply:
Yup commits are sorted by date by default .. We are fetching the commits in descending order of date.. The latest commit is on the top check developer-interface-ros.herokuapp.com/api/commits

Reply:
If the builds are triggered by hand, it will be difficult to keep track of them i guess.. Anyway we can't load the whole data into memory, so this seems the only way